### PR TITLE
Add USB VID/PID for PewPew boards to the CircuitPython mode

### PR DESCRIPTION
**Pull request by deshipu**
I won't be using Adafruit's VID/PID indefinitely.

### CODE REVIEW

File: mu/modes/circuitpython.py [hunk 1]
```diff
@@ -48,6 +48,7 @@ class CircuitPythonMode(MicroPythonMode):
         (0x1B4F, 0x8D23),  # SparkFun SAMD21 Dev Breakout
         (0x1209, 0x2017),  # Mini SAM M4
         (0x1209, 0x7102),  # Mini SAM M0
+        (0x1D50, 0x801D),  # PewPew Boards
```
Comment (carlosperate) [inline mu/modes/circuitpython.py]: I've only found the `0x60e8` VID in https://github.com/openmoko/openmoko-usb-oui/blob/master/usb_product_ids.psv, is there a different source for this list or has the VID been reserved some other way?

Comment (deshipu) [reply mu/modes/circuitpython.py]: Och, you are right I have no idea how I made that mistake, thanks for catching it!

Comment (deshipu) [reply mu/modes/circuitpython.py]: I see what I did, I copied the Adafruit VID instead of mine. Thanks!

**Comment by dybber** [comment]
Thanks for the contribution! I've made the necessary changes to be able to merge this and made a new pull request at #1094, and I'm thus closing this PR.

**Pull request closed without merging**